net/lp_rpc.proto

```diff
@@ -160,8 +164,14 @@ message OrchestratorInfo {
   // Data for transcoding authentication
   AuthToken auth_token = 6;
 
+  // Information on Orchestrator hardware
+  bytes hardware = 7;
```

Comment:
Why is it byte, shouldn't we give it some structure?

Reply:
I used bytes in the grpc part to try and limit updates needed to the proto files.  go-livepeer has structs to parse the data when it is received by the Orchestrator

Reply:
The `hardware` field is json bytes of the structs defined in `ai/worker/runner.gen.go`.  

Was just trying to avoid having two auto generated go structs so I converted it to bytes before sending through grpc.

I can also add these as new message definitions in proto file if prefer that.  

https://github.com/livepeer/go-livepeer/blob/9a2743592d7c8f299367a03dfb3bdfbb1c351fd8/ai/worker/runner.gen.go#L208-L215

https://github.com/livepeer/go-livepeer/blob/9a2743592d7c8f299367a03dfb3bdfbb1c351fd8/ai/worker/runner.gen.go#L239-L243